### PR TITLE
Create list of all branches page

### PR DESCRIPTION
**Pull request by IrinaEndeladze**

### CODE REVIEW

**Comment by NikitoriusMinsk** [comment]
I think you renamed the main branch, please don't pull requests this big

File: components/ui/Pagination.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    totalCount,
+    siblingCount,
+    pageSize,
+  });
+
+  if (currentPage === 0 || paginationRange.length < 2) {
+    return null;
+  }
+
+  const onNext = () => {
```
Comment (NikitoriusMinsk) [inline components/ui/Pagination.js:26]: add a check, so the pagination counter doesnt go over max amount of pages

File: components/ui/Pagination.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    onPageChange(currentPage + 1);
+  };
+
+  const onPrevious = () => {
```
Comment (NikitoriusMinsk) [inline components/ui/Pagination.js:30]: add a check, so the pagination counter doesnt go over min amount of pages

File: components/ui/Pagination.js [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+          alt="arrow left"
+        />
+      </li>
+      {paginationRange.map((pageNumber) => {
+        if (pageNumber === DOTS) {
+          return <li className={s.paginationItemDots}>&#8230;</li>;
+        }
+
+        return (
+          // eslint-disable-next-line react/jsx-key
```
Comment (NikitoriusMinsk) [inline components/ui/Pagination.js:51]: add keys, so react knows which component to update on state change

File: pages/clinicsPage/index.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import ClinicPageCardList from "../../components/contents/ClinicPageCardList";
+function ClinicsPage() {
+  return (
+    <>
+      <ClinicPageCardList />
```
Comment (NikitoriusMinsk) [inline pages/clinicsPage/index.js:5]: move ClinicPageCardList into the page file

**Review by NikitoriusMinsk — CHANGES_REQUESTED** [review]

File: pages/clinicsPage/index.js (hunk 4 above)
Comment (IrinaEndeladze) [reply pages/clinicsPage/index.js:5]: it is in pages file

Comment (NikitoriusMinsk) [reply pages/clinicsPage/index.js]: i mean the component's code

File: pages/clinicsPage/index.js [hunk 5]
```diff
@@ -1,10 +1,6 @@
 import ClinicPageCardList from "../../components/contents/ClinicPageCardList";
 function ClinicsPage() {
-  return (
-    <>
-      <ClinicPageCardList />
```
Comment (NikitoriusMinsk) [inline pages/clinicsPage/index.js]: check pull request comments

**Review by NikitoriusMinsk — CHANGES_REQUESTED** [review]

**Review by NikitoriusMinsk — APPROVED** [review]

**Pull request merged**